.github/workflows/release.yaml

```diff
@@ -0,0 +1,18 @@
+on:
+  push:
+    branches:
+      - main
+
+permissions:
+  contents: write
+  pull-requests: write
+
+name: release-please
+
+jobs:
+  release-please:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: googleapis/release-please-action@v4
```

Comment:
I think you have to specify the mod file. In this repo we have multiple modules. We only want to release v2.